pom.xml

```diff
@@ -148,7 +148,7 @@
     <vers.hbase2>0.98.3-hadoop2</vers.hbase2>
     <vers.hbase-cdh4>0.94.6-cdh${cdh4.version}</vers.hbase-cdh4>
     <vers.hbase-cdh5>0.96.1.1-cdh${cdh5.version}</vers.hbase-cdh5>
-    <vers.hive>0.10.0</vers.hive>
+    <vers.hive>0.13.0</vers.hive>
```

Comment:
I'm wary of updating the default Hive version so many versions. What changed that requires that we move to 0.13?